pages/order-list/index.js

```diff
@@ -1 +1,20 @@
-export default function OrderList() {}
+import Drinks from "../components/order-menu/drink-menu";
+import Maindishes from "../components/order-menu/main-menu";
+
+export default function OrderList() {
+  return (
+    <>
+      <h1>Order Menu</h1>
+      <form>
```

Comment:
Why a ``<form>``?

Reply:
I thought maybe to put the main dishes in a separate container and the drinks also so you can swipe the different menu pages at some point

Reply:
A Form is just used, if there are some inputs where the user can type in some data and you need to collect that data. This is not the case here. So <form> has no business here :)